examples/opensora_pku/README.md

```diff
@@ -342,30 +313,22 @@ python tools/ckpt/combine_ckpt.py --src output_dir/ckpt --dest output_dir/ckpt -
 Afterwards, you will obtain a full checkpoint file under `output_dir/ckpt/rank_0/full_0.ckpt`.
 > If you want to run inference with this full checkpoint file, please revise the script `scripts/text_condition/sample_video.sh` and append `--pretrained_ckpt output_dir/ckpt_full/rank_0/full_0.ckpt` to the end of the inference command.
 
-Then please revise `scripts/text_condition/train_videoae_65x256x256.sh`, and change `--pretrained` to the full checkpoint path from the `17x256x256` stage. Then run:
+Then please revise `scripts/text_condition/train_videoae_221x512x512.sh`, and change `--pretrained` to the full checkpoint path from the `65x512x512` stage. Then run:
 
 ```bash
-# start 65x256x256 finetuning, 8 NPUs
-bash scripts/text_condition/train_videoae_65x256x256.sh
-```
-Simiarly, please revise the `--pretrained` to the full checkpoint path from the `65x256x256` stage, and then start the third-stage training with:
-
-```bash
-# start 65x512x512 finetuning, 8 NPUs
-bash scripts/text_condition/train_videoae_65x512x512.sh
+# start 221x512x512 finetuning, 8 NPUs
+bash scripts/text_condition/train_videoae_221x512x512.sh
 ```
+Simiarly, please revise the `--pretrained` to the full checkpoint path from the `221x512x512` stage, and then start the third-stage training (to be released soon).
 
 #### Performance
 
 We evaluated the training performance on MindSpore and Ascend NPUs. The results are as follows.
 
 | Model           | Context        | Precision | BS | NPUs | num_frames + num_images| Resolution  | Train T. (s/step) |
 |:----------------|:---------------|:----------|:--:|:----:|:-----------:|:-----------:|:--------------:|
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 4  |  8   |   17 + 4    | 256x256     |   1.8     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 4  |  8   |   65 + 4    | 256x256     |   4.5     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 2  |  8   |   17 + 4    | 512x512     |   3.6     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 4  |  8   |   17 + 4    | 512x512     |   7.5     |
-| LatteT2V-XL/122 | D910\*x1-MS2.3 | FP16      | 2  |  8   |   65 + 16   | 512x512     |   16.8    |
+| LatteT2V-XL/122 | D910\*x1-MS2.3 | BF16      | 2  |  8   |   17 + 4    | 512x512     |       |
+| LatteT2V-XL/122 | D910\*x1-MS2.3 | BF16      | 2  |  8   |   65 + 16   | 512x512     |       |
```

Comment:
MS2.3 master(0615daily)

Reply:
https://repo.mindspore.cn/mindspore/mindspore/version/202406/20240615/

Better to add this hyperlink

Reply:
CANN C18 (0517)

Reply:
Done